airflow/providers/amazon/aws/hooks/glue_crawler.py

```diff
@@ -44,6 +45,10 @@ def glue_client(self):
         """:return: AWS Glue client"""
         return self.get_conn()
 
+    @cached_property
+    def sts_hook(self):
+        return StsHook(aws_conn_id=self.aws_conn_id)
+
```

Comment:
IMHO, better not to add one hook property to another if it possible or at least make it "private"
Looks like we only need to call it once in GlueCrawlerHook, so we could directly call in `update_tags` method

```python
account_id = StsHook(aws_conn_id=self.aws_conn_id).get_account_number()
```

As long term (separate PR) it is a good idea to create `account_id` property in `AwsBaseHook`.